### ISSUE TITLE
CLI glitches on Windows + Git Bash

### ISSUE THREAD

**Issue by 3dbrows** [infracost/infracost#1265]
```
$ infracost -v
Infracost v0.9.16
```

Using `Git Bash 2.28.0.windows.1` on Windows 10 (a common way to use Git, and common Linux CLI utilities) produces the following output. The program quits immediately with no opportunity to supply a y/N answer.
```
$ infracost register
You already have an Infracost API key saved in C:\Users\Duncan\.config\infracost/credentials.yml
? Would you like to generate a new API key? [y/N] Ôûê

Error: ^D
```

This is avoided by using `winpty`, but on entering `y` some output is doubled.
```
$ winpty infracost register
You already have an Infracost API key saved in C:\Users\Duncan\.config\infracost/credentials.yml
Would you like to generate a new API key: y
Would you like to generate a new API key: y
```

Of note: `kubectl exec` detects and objects to being run without `winpty` in Git Bash:
```
$ kubectl exec -it ubuntu-846b56b68c-qtfxn -- sh
Unable to use a TTY - input is not a terminal or the right kind of file
```
```
$ winpty kubectl exec -it ubuntu-846b56b68c-qtfxn -- sh
#
```

**Comment by vdmgolub** [comment]
This is fixed and will be released with v0.9.19.

**Comment by tim775** [comment]
@3dbrows This is live in v0.9.19
